eng/source-build-release-announcement.md

```diff
@@ -7,4 +7,13 @@ Please use the [$TAG tag]($TAG_URL) to source-build .NET version $RUNTIME_VERSIO
 - Release Notes: $RELEASE_NOTES_URL
 - Tag URL: $TAG_URL
 
-@dotnet/distro-maintainers
\ No newline at end of file
+@dotnet/distro-maintainers
+
+<!--
+{
```

Comment:
I agree with @crummel that a better indication of the machine readable section should be considered.  Avoiding a comment block would make this more visible and promote usage.  e.g. ```json would be better IMO.

Reply:
I see, would we rather have the info be visible in the announcement? The original suggestion was explicitly for a comment.

I can put together an example of how to pull the info from the discussion announcement.